games-puzzle/monsterz_game/monsterz-0.7.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A little arcade puzzle game"
+DESCRIPTION="The goal of the game is to create rows of similar monsters, either horizontally or vertically.\
+The only allowed move is the swap of two adjacent monsters, on the condition that it creates a row of three \
+or more. When alignments are cleared, pieces fall from the top of the screen to fill the board again. Chain \
+reactions earn you even more points."
+HOMEPAGE="http://sam.zoy.org/monsterz/"
+COPYRIGHT="2007 Sam Hocevar, Brendan Rackley, MenTaLguY, Sun Microsystems\
+Michael Speck, David White Mike Kershaw"
+LICENSE="WTFPL
```

Comment:
```diff
-LICENSE="WTFPL
+LICENSE="WTFPL v2
```
(This is because you are adding the license as "WTFPL v2", not "WTFPL".)

BTW, It's a very good idea to add the " v2".

Regarding the `SOURCE_URI`, I'm quite sure it is recommended to use http or, even better, https, instead of svn.